components/aks/init.tf

```diff
@@ -54,6 +54,9 @@ locals {
       subscription = "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
     }
 
+    demo = {
+      subscription = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```

Comment:
I dont think this is the correct subscription ID?

Reply:
corrected